README.md

```diff
@@ -147,6 +156,14 @@ The distributed snapshotting feature is provided to handle snapshot operations f
 
 Other than this, the NODE_NAME environment variable must be set where the CSI snapshotter sidecar is deployed. The value of NODE_NAME should be the name of the node where the sidecar is running.
 
+### Volume Group Snapshot Support
+
+The following requisites must be met to unable the volume group snapshot feature:
+
+* the Volume Group Snapshot CRDs are installed in the cluster
+* the `--enable-volume-group-snapshots=true` option is being passed to the snapshot controller
```

Comment:
specifically, deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml needs to be updated with that flag in order to enable this feature in the snapshot controller.